doc/manual/manual.tex

```diff
@@ -1339,21 +1354,234 @@ \subsection{Numerical methods}
+approximation (ALA), truncated anelastic liquid approximation (TALA), and the
+Boussinesq approximation (BA). These can all be derived from the basic
+equations~\eqref{eq:stokes-1}--\eqref{eq:temperature} via various approximations, 
+and we will discuss them in the following. Since they are typically only provided
+considering velocity, pressure and temperature, we will in the following omit
+the dependence on the compositional fields used in previous sections, though
+this dependence can easily be added back into the equations stated below. A
+detailed discussion of the approximations introduced below can also be found in
+\cite{KLKLZTTK10}.
+
+The three approximations mentioned all start by writing the pressure and
+temperature as the sum of a (possibly depth dependent)
+reference state plus a perturbation, i.e., we will write
+\begin{align*}
+  p(\mathbf x,t) &= \bar p(z) + p'(\mathbf x,t),
+  \\
+  T(\mathbf x,t) &= \bar T(z) + T'(\mathbf x,t).
+\end{align*}
+Here, barred quantities are reference states and may depend on the depth $z$
+(not necessarily the third component of $\mathbf x$) whereas primed quantities
+are the spatially and temporally variable deviations of the temperature and
+pressure fields from this reference state. In particular, the reference pressure
+is given by solving the hydrostatic equation,
+\begin{align}
+\label{eq:hydrostatic-pressure}
+  \nabla \bar p = \bar\rho \mathbf g,
+\end{align}
+where $\bar\rho=\rho(\bar p,\bar T)$ is a \textit{reference density} that
+depends on depth and represents a typical change of material parameters and solution 
+variables with depth. $\bar T(z)$ is chosen as an adiabatic profile accounting for the
+fact that the temperature increases as the pressure increases.
+With these definitions, equations \eqref{eq:stokes-1}--\eqref{eq:stokes-2} can equivalently be written as follows:
+\begin{align}
+  \label{eq:stokes-decomposed-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  (\rho-\bar\rho) \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-decomposed-2}
+  \nabla \cdot (\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+Likewise, when omitting entropic effects, the temperature equation reads as
+\begin{multline}
+  \label{eq:temperature-decomposed}
+  \rho C_p \left(\frac{\partial T}{\partial t} + \mathbf u\cdot\nabla T\right)
+  - \nabla\cdot k\nabla T
+  \\
+  =
+  \rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha T \left( \mathbf u \cdot \nabla p \right)
+  \quad
+  \textrm{in $\Omega$},
+\end{multline}
+where the right-hand side includes radiogenic heat production, shear heating and adiabatic heating (in that order).
```

Comment:
how does this equation differ from the original one at all? (Other than the omission of entropic heating?)